### ISSUE TITLE
Support for UPPER_CASE arrays

### ISSUE THREAD

**Issue by gylaz** [edvardchen/eslint-plugin-i18next#12]
Is there a way to ignore array constants with UPPER_CASE names?

Example:
```js
const MY_ARRAY = [
  'foo',
  'bar',
  'baz'
];
```

The above will trigger the lint warning/error. Is this by design? Any plans to support ignoring arrays as well?

**Comment by edvardchen** [comment]
`eslint` helps people to write less error-prone codes but doesn't prevent people from "runtime unexpected behaviors"

Sure. We should allow this, including the following cases:
```js
const a = {
  MY_ARRAY: ['foo', 'bar']
  [MY_ARRAY]: ['foo', 'bar']
}
```

I would work on it.
